weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableQuery.ts

```diff
@@ -131,6 +131,59 @@ export const useCallsForQuery = (
   }, [callResults, calls.loading, total, costs.loading, costResults, refetch]);
 };
 
+export const useCallsForQueryCharts = (
```

Comment:
this does not have anything "Charts" specific, it is more of a refactor of the first section of `useCallsForQuery`. I'd recommend renaming this and adding a comment that it should be merged with `useCallsForQuery` (assuming you don't want to do that (: )